assets/js/blocks/cart-checkout/cart/full-cart/index.js

```diff
@@ -22,71 +22,92 @@ import placeholderShippingMethods from '../../placeholder-shipping-methods';
 import './style.scss';
 
 /**
- * Component that renders the Cart block when user has something in cart aka "full".
+ * Given an API response with cart totals, generates an array of rows to display in the Cart block.
+ *
+ * @param {Object} cartTotals - Cart totals data as provided by the API.
+ * @returns {Object[]} Values to display in the cart block.
  */
-const Cart = () => {
-	// @todo this are placeholders
-	const onActivateCoupon = ( couponCode ) => {
-		// eslint-disable-next-line no-console
-		console.log( 'coupon activated: ' + couponCode );
-	};
-	const cartTotals = {
-		currency: 'EUR',
-		currency_minor_unit: 2,
-		total_items: 6000,
-		total_items_tax: 0,
-		total_fees: 0,
-		total_fees_tax: 0,
-		total_discount: 0,
-		total_discount_tax: 0,
-		total_shipping: 0,
-		total_shipping_tax: 0,
-		total_tax: 0,
-		total_price: 6000,
-	};
-
-	const currency = getCurrencyFromPriceResponse( cartTotals );
-	const totalItems = [
+const getTotalRows = ( cartTotals ) => {
+	const totalItems = parseInt( cartTotals.total_items );
+	const totalItemsTax = parseInt( cartTotals.total_items_tax );
+	const totalRows = [
```

Comment:
You are using `parseInt` for converting all the values in this new function.  Wherever, `parseInt` is being used should you also be including the `radix` value?  It doesn't default to `10` so we probably should be explicit here.

Reply:
I thought that shouldn't affect us since ES5 removed octal interpretation and defaults to decimal: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/parseInt#ECMAScript_5_removes_octal_interpretation.

But yes, added the radix to make it explicit in 9755e7bca1c22c04522a737d744bde6f7abb7e89. Should we add an [eslint rule](https://eslint.org/docs/rules/radix) in a follow-up to make sure we are consistent with this?

Reply:
The MDN documentation has a [callout explicitly warning against assuming radix defaults to 10](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/parseInt#Syntax).  While the documentation also references the ES5 standard, and our browser support range implements that standard, it still does not default to 10 and with that ambiguity I think it's important to be explicit. So thanks for making that change :)

> Should we add an eslint rule in a follow-up to make sure we are consistent with this?

Ya that's a good idea.